src/auth.rs

```diff
@@ -0,0 +1,63 @@
+use diesel::{prelude::*, result::Error::NotFound};
+use rocket::{
+    http::Status,
+    request::{self, FromRequest, Outcome, Request},
+};
+
+use crate::{
+    models::{token::Token, user::User},
+    DbConn,
+};
+
+pub struct HaaSUser(pub User);
```

Comment:
```suggestion
pub struct HaasUser(pub User);
```

Follow rust style conventions